src/main/java/uk/gov/hmcts/probate/transformer/CallbackResponseTransformer.java

```diff
@@ -680,10 +680,17 @@ public String setExecutorNames(CaseData caseData,
             && caseData.getSolsWillType().matches("WillLeft")) {
             executorNames = "The executor" + returnPlural(listOfApplyingExecs) + " ";
 
-            executorNames = listOfApplyingExecs.isEmpty() ? executorNames + professionalName + ": " :
-                executorNames + FormattingService.createExecsApplyingNames(listOfApplyingExecs)
-                    +  ", " + caseData.getPrimaryApplicantForenames()
-                    + " " + caseData.getPrimaryApplicantSurname() + ": ";
+            if (caseData.getSolsSolicitorIsExec().matches(YES) && caseData.getSolsSolicitorIsApplying().matches(YES)
+                || (caseData.getTitleAndClearingType().matches("TCTTrustCorpResWithSDJ")
+                || caseData.getTitleAndClearingType().matches("TCTTrustCorpResWithApp"))) {
```

Comment:
these last 2 of code lines can be simplified, and reuse code more by using:
getTrustCorpTitleClearingTypes().contains(caseData.getTitleAndClearingType())

getTrustCorpTitleClearingTypes is in model.Constants